configs/alibaba-inc_clarity_design.json

```diff
@@ -14,9 +14,27 @@
       ]
     },
     {
-      "url": "https://at-design.alibaba-inc.com/doc/layout-pattern/",
+      "url": "https://at-design.alibaba-inc.com/basic/",
       "tags": [
-        "doc"
+        "basic"
+      ]
+    },
+    {
+      "url": "https://at-design.alibaba-inc.com/biz/",
+      "tags": [
+        "biz"
+      ]
+    },
+    {
+      "url": "https://at-design.alibaba-inc.com/uiless/",
+      "tags": [
+        "uiless"
+      ]
+    },
+    {
+      "url": "https://at-design.alibaba-inc.com/resource/",
```

Comment:
DocSearch is only provided for documentation web pages. Please remove the start_urls item not related to documentation (namely: biz, resource, uiless)

Reply:
How to define documentation?

Reply:
> DocSearch is only provided for documentation web pages. Please remove the `start_urls` item not related to documentation (namely: biz, resource, uiless)

I add sitemap_urls field in my config and  update the start_urls as you say, please check them out, thx a lot ^_^

Reply:
Thanks @Mavericker-1996

Documenation here means technical documentation that is helpful for the community.

As stated in [our checklist](https://community.algolia.com/docsearch/who-can-apply.html):
> Your website must be a **documentation website**. We do not index blogs or commercial content.

Please remove the `start_urls` item not related to documentation (namely: biz, resource, uiless)

Thanks

Reply:
I have removed the resource in the `start_urls`，but both [biz](https://at-design.alibaba-inc.com/biz/at-text-avatar) & [uiless](https://at-design.alibaba-inc.com/uiless/uiless-extendable-nav) pages are documentation as you mentioned :)

Reply:
Thanks @Mavericker-1996

The pages you have pointed me out are extremely long to render. The loading of these pages ends with a blank page. Can you let us know once these pages are available? Thanks

Reply:
![image](https://user-images.githubusercontent.com/30346283/67561871-35965600-f750-11e9-8a0d-f9db1a85b477.png)
![image](https://user-images.githubusercontent.com/30346283/67561902-45ae3580-f750-11e9-9aca-2b2223e3e2da.png)

As you can see, we are a  React UI library website, both [biz](https://at-design.alibaba-inc.com/biz/at-text-avatar) & [uiless](https://at-design.alibaba-inc.com/uiless/uiless-extendable-nav) pages are components documentation. you can them out in the screenshots aboved. Hoping you can be available to these pages :)

Reply:
Thanks for the details. I will merge it and deploy it but please be aware that due to the rendering being slow/not complete, the crawl might be partial. Please make sure you either avoid every client side rendering (JS rendering) or make sure the time need to render them is shorter than 1 second.